### DISCUSSION TITLE
Zero Result Values

### DISCUSSION THREAD

**Discussion by sajaddp** [laravel/nova-issues#5725] [Q&A]
Hi,
Can someone explain this part to me? 😅

https://nova.laravel.com/docs/4.0/metrics/defining-metrics.html#zero-result-values

_By default, Nova will handle results of 0 as a result containing no data. This may not always be correct, which is why you can use the allowZeroResult method to indicate that 0 is a valid value result:_
```php
return $this->result(0)->allowZeroResult();
```
Where should it be applied?

**Comment by davidhemphill** [comment]
That should be called in your metric's `calculate` method after returning a result. Does that make sense?
